app/graphql/mutations/pool_samples_mutation.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Mutations
+  # Mutation to handle the creation of a library(s), by pooling samples
+  class PoolSamplesMutation < BaseMutation
+    argument :arguments, Types::Inputs::PoolSamplesInputType, required: true
+
+    field :library, [Types::Outputs::LibraryType], null: true
+    field :errors, [String], null: false
+
+    # def resolve(arguments:)
+    #   debugger
```

Comment:
- create tube with barcode
- create library
- add tag to existing request
- link request with library